Library/Homebrew/cmd/which.rb

```diff
@@ -0,0 +1,17 @@
+module Homebrew
+  def which
+    cmd = ARGV.first
+
+    if HOMEBREW_INTERNAL_COMMAND_ALIASES.has_key? cmd
+      cmd = HOMEBREW_INTERNAL_COMMAND_ALIASES[cmd]
+    end
+
+    if (path = HOMEBREW_LIBRARY_PATH/"cmd/#{cmd}.rb"; path.file?)
```

Comment:
Although, I do wonder should we implement `bfontaine/homebrew-aliases` inside the core so `brew which` can print the alias content rather than file.